concepts/05 UI Components/Toast/00 Getting Started with Toast/05 Display a Toast/10 Show and Hide the Toast.md

```diff
@@ -0,0 +1,173 @@
+When you need to show a notification in the Angular, Vue, or React, you can use one of the **notify** methods, described in the section above. But if you need to customize the toast, use a different technique. Bind the [visible](/api-reference/10%20UI%20Components/dxOverlay/1%20Configuration/visible.md '/Documentation/ApiReference/UI_Components/dxToast/Configuration/#visible') property of the Toast component to a component (Button in the example) property. After that, change this property, and the Toast will appear or disappear.
```

Comment:
It's not clear what additional customizations are enabled if I use the property-binding technique. You say "if you need to customize the toast"... But the demo seems to set the same options that I can set in the `notify` method. Can you clarify that part - what is the benefit compared to those methods? 

Update: Now that I read the following section, I think that maybe you meant "But if you need to customize Toast content".